mmj2jar/PATutorial/Page402.mmp

```diff
@@ -1,27 +1,25 @@
 $( <MM> <PROOF_ASST> THEOREM=reiteration  LOC_AFTER=
-*                                                          Page402.mmp
-h1::reiteration.1  |- ph
-qed::              |- ph
+*                                                          Page403.mmp
```

Comment:
I guess the `.mmp` names should be fixed here and in Page403.mmp

Reply:
Yes indeed. Will fix.